packages/react/src/progress/Progress.css.ts

```diff
@@ -0,0 +1,38 @@
+import { theme } from "@optiaxiom/globals";
+
+import { recipe, type RecipeVariants, style } from "../vanilla-extract";
+
+export const progress = recipe({
```

Comment:
let's call this `indicator` since the outer component is the actual progress component/track
```suggestion
export const indicator = recipe({
```